src/shared/DotnetTool/DotnetTool.csproj

```diff
@@ -9,6 +9,7 @@
       publishDir=$(PublishDir);
     </NuspecProperties>
     <NoBuild>true</NoBuild>
-    <OutputPath>../../../out/nupkg</OutputPath>
+    <OutputPath>$(ProjectOutPath)nupkg\$(Configuration)\</OutputPath>
+    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
```

Comment:
This just prevents MSBuild from creating the empty/not used `net6.0` directory under `$(OutputPath)/`.